### ISSUE TITLE
Range error when conditioning on initial tree

### ISSUE THREAD

**Issue by nspope** [scrm/scrm#130]
Setting initial trees from a separate file is broken:

```
git clone https://github.com/scrm/scrm.git
cd scrm && ./bootstrap && make
echo '((1:1.0,2:1.0):1.0,3:1.0);' >init.nwk
./scrm 3 1 -init init.nwk
```
==> `Error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)`.

This is thrown from within `readNewick()` by https://github.com/scrm/scrm/blob/4e78eb73fb026677fc99eb097aaa95dd7a3fa8c6/src/forest.cc#L1340

It seems that the vector storing the recombination breakpoints is not initialized properly in `readNewick()` and that results in a range error when `current_base()` is called. Changing the following line:

https://github.com/scrm/scrm/blob/4e78eb73fb026677fc99eb097aaa95dd7a3fa8c6/src/forest.cc#L1327

to:
```
this->set_next_base(0.0);
++current_rec_;
```

in analogy to `buildInitialTree()` seems to fix this error (at least, it results in a reasonable sequence of trees that begins from the desired initial state).

**Comment by paulstaab** [comment]
This is amazing! THANK YOU for reporting this bug and doing all the work to fix it! I'll merge your fix as #132.

**Comment by paulstaab** [comment]
Should be fixed in new version 1.7.4
